middle_end/flambda2/tests/api_tests/dune

```diff
@@ -1,10 +1,17 @@
 (tests
  (names extension_meet)
  (modes native)
- (instrumentation (backend bisect_ppx))
- (flags (:standard -principal))
+ (instrumentation
+  (backend bisect_ppx))
```

Comment:
This is one of several files that seems to have been reformatted by an editor. Have we defined a style for dune files somewhere? If not, the reformatting should be left for a different PR.